include/sound/sof/topology.h

```diff
@@ -45,6 +45,13 @@ enum sof_comp_type {
 	SOF_COMP_FILEWRITE = 10001,	/**< host test based file IO */
 };
 
+/* Scheduling domain, unset, Low Latency, or Data Processing */
+enum sof_comp_domain {
+	SOF_COMP_DOMAIN_UNSET = 0,
+	SOF_COMP_DOMAIN_LL,
+	SOF_COMP_DOMAIN_DP,
+};
```

Comment:
I would 
1) add a comment here that by default LL is used if no domain is specified.
2) remove UNSET
3) assign LL to 0, DP to 1.

Reply:
I need unset or similar value to indicate the default behaviour of getting the value from the manifest.  Comments added.